docs/data-version-control.md

```diff
@@ -0,0 +1,22 @@
+## Data version control for MindGPT
```

Comment:
Should this page be updated to explain the process in setting up a new remote storage and connecting it to DVC for people without access to our Azure bucket?

Reply:
Fleshed this out some more, so someone could set up their own.